src/contexts/AuthContext.tsx

```diff
@@ -73,6 +79,24 @@ export const AuthProvider: React.FC<IProps> = ({ children }: IProps) => {
     },
     [onAuthenticated]
   );
+  const onRequestPasswordReset = useCallback(async (params: IResetPasswordParams) => {
+    const fn = UserAPI.requestResetPassword;
+    try {
+      const user = await fn(params);
```

Comment:
is the response actually a `user`? do you even care about it? you're not using it anywhere

Reply:
removed this logic